### PR TITLE
chore: Update translations

### PR DESCRIPTION
**Pull request by RuffleBuild**
New Crowdin pull request with translations 🎉

### CODE REVIEW

File: desktop/assets/texts/es-ES/preferences_dialog.ftl [hunk 1]
```diff
@@ -28,5 +28,5 @@ gamemode = GameMode
 gamemode-tooltip =
     GameMode aplica temporalmente un set de optimizaciones a tu computadora y/o a Ruffle.
     Ruffle solicita GameMode solo cuando se está reproduciendo una película.
-gamemode-default = Por defecto
-gamemode-default-tooltip = GameMode solo será habilitado cuando la preferencia de poder esté en alto.
+gamemode-default = gamemode-predeterminado
```
Comment (torokati44) [inline desktop/assets/texts/es-ES/preferences_dialog.ftl:31]: This is definitely not right...

**Review by torokati44 — CHANGES_REQUESTED** [review]

File: desktop/assets/texts/es-ES/preferences_dialog.ftl (hunk 1 above)
Comment (danielhjacobs) [reply desktop/assets/texts/es-ES/preferences_dialog.ftl:31]: They translated the key instead of translating the value...

Comment (torokati44) [reply desktop/assets/texts/es-ES/preferences_dialog.ftl:31]: Undid it on Crowdin.

**Comment by torokati44** [comment]
Summoning @sombraguerrero, because I saw that you approved these after "Load3r" suggested them (with one exception, after undoing that translated string key mistake).

At least one of these strings having changed in https://github.com/ruffle-rs/ruffle/pull/18839, and is now essentially reverted. I think it would be great to avoid any sort of back-and-forth between the variants in the future.

**Comment by sombraguerrero** [comment]
I acknowledge my timing on that one sucked. Just haven't had a to of time lately, sorry.

**Comment by torokati44** [comment]
Yeah no worries - I manly wanted confirmation about the rest of the strings, that these are indeed the better ones, and are here to stay. 😊

File: web/packages/core/texts/ja-JP/messages.ftl [hunk 2]
```diff
@@ -16,6 +16,14 @@ view-error-details = エラーの詳細を表示
...
 click-to-unmute = クリックでミュートを解除
 clipboard-message-title = コピーしてRuffleに貼り付け
+clipboard-message-description =
+    { $variant ->
+       *[unsupported] お使いのブラウザはクリップボードへのフルアクセスをサポートしていません。
+        [access-denied] クリップボードへのアクセスが拒否されました。
+    } ですが、以下のショートカットを利用できます:
+clipboard-message-copy = { " " } からコピー
+clipboard-message-cut = { " " } : 切り取り
+clipboard-message-paste = { " " } : 貼り付け
```
Comment (torokati44) [inline web/packages/core/texts/ja-JP/messages.ftl:26]: It's strange that these are not consistent...

Comment (creeper-0910) [reply web/packages/core/texts/ja-JP/messages.ftl]: This is my mistake, sorry.
I will correct it later.

File: desktop/assets/texts/fr-FR/settings.ftl [hunk 3]
```diff
@@ -70,7 +70,7 @@ scale-mode-force = Forcer
 scale-mode-force-tooltip = Empêche le film de changer de mode de mise à l'échelle, en le verrouillant sur le paramètre sélectionné.
 player-version = Version du lecteur
 player-runtime = Type de lecteur
-player-runtime-flash = Flash Player
+player-runtime-flash = Lecteur Flash
```
Comment (kjarosh) [inline desktop/assets/texts/fr-FR/settings.ftl:73]: Should this be translated? Adobe doesn't translate runtime names in their docs. I see that it's translated in some languages, in some it isn't

File: desktop/assets/texts/fr-FR/file_picker.ftl [hunk 4]
```diff
@@ -1,6 +1,6 @@
 file-picker-title-open-file = Ouvrir un seul fichier
 file-picker-filter-supported = Tous les formats pris en charge
 file-picker-filter-swf = SWF (*.swf)
-file-picker-filter-spl = FutureSplash Animator (*.spl)
+file-picker-filter-spl = FutureSplash Animateur (*.spl)
```
Comment (kjarosh) [inline desktop/assets/texts/fr-FR/file_picker.ftl:4]: This seems wrong too, it's a name of the product

Comment (kjarosh) [reply desktop/assets/texts/fr-FR/file_picker.ftl:4]: Reverted on Crowdin

File: desktop/assets/texts/fr-FR/settings.ftl (hunk 3 above)
Comment (Toad06) [reply desktop/assets/texts/fr-FR/settings.ftl:73]: Good catch.☺️I did almost all of the French translation, and I agree that this shouldn't be translated in this context.

I'm also unsure about the change related to the letterbox setting: while the translation itself is correct, the word "letterbox" is usually not translated (eg. https://fr.wikipedia.org/wiki/Letterbox).

Comment (kjarosh) [reply desktop/assets/texts/fr-FR/settings.ftl:73]: Reverted this, also updated translations for other languages where this shouldn't be translated

File: desktop/assets/texts/es-ES/preferences_dialog.ftl [hunk 5]
```diff
@@ -28,5 +28,5 @@ gamemode = GameMode
 gamemode-tooltip =
     GameMode aplica temporalmente un set de optimizaciones a tu computadora y/o a Ruffle.
     Ruffle solicita GameMode solo cuando se está reproduciendo una película.
-gamemode-default = Por defecto
-gamemode-default-tooltip = GameMode solo será habilitado cuando la preferencia de poder esté en alto.
+gamemode-default = Predeterminado
```
Comment (torokati44) [inline desktop/assets/texts/es-ES/preferences_dialog.ftl:31]: Are you sure that this is better, @kjarosh?

File: desktop/assets/texts/de-DE/settings.ftl [hunk 6]
```diff
@@ -3,6 +3,7 @@ player-settings = Spieler-Einstellungen
 movie-parameters = Filmparameter
 custom-base-url = Benutzerdefinierte Base-URL
 spoof-swf-url = Gefälschte SWF-URL
+referer-url = Referenz URL
```
Comment (torokati44) [inline desktop/assets/texts/de-DE/settings.ftl:6]: Hmm: https://de.wikipedia.org/wiki/Referrer

Which one do you think is more appropriate, @evilpie?

File: desktop/assets/texts/de-DE/settings.ftl [hunk 7]
```diff
@@ -45,7 +46,28 @@ align-top-right = Oben-Rechts
 align-bottom-right = Unten-Rechts
 align-force = Erzwingen
 scale-mode = Skalierungsmodus
+scale-mode-noscale = Unskaliert (100%)
+scale-mode-noscale-tooltip =
+    Zeigt den Film in der Originalgröße an, ohne Zoom.
```
Comment (torokati44) [inline desktop/assets/texts/de-DE/settings.ftl:51]: There's a period at the end of this sentence, but not at the end of the ones for the other scale modes.

File: desktop/assets/texts/es-ES/preferences_dialog.ftl (hunk 5 above)
Comment (kjarosh) [reply desktop/assets/texts/es-ES/preferences_dialog.ftl:31]: I'm not sure, it's not _obviously_ wrong for me, so I trusted the translator

File: desktop/assets/texts/de-DE/settings.ftl (hunk 7 above)
Comment (kjarosh) [reply desktop/assets/texts/de-DE/settings.ftl:51]: Fixed

**Comment by kjarosh** [comment]
@torokati44 I think we should merge this PR and leave remaining concerns as comments/issues on Crowdin. They are not blocking and some translations have been waiting for over 27 days as of now. What's your opinion on this?

**Pull request merged**